pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -145,6 +145,7 @@ in (buildEnv {
     # patch texmf-dist  -> $TEXMFDIST
     # patch texmf-local -> $out/share/texmf-local
     # patch texmf.cnf   -> $TEXMFSYSVAR/web2c/texmf.cnf
+    # add TEXMFAUXTREES to texmfcnf.lua to support setup hooks
```

Comment:
Have you tested whether that works? If I remember correctly, ConTeXt only uses its file database for lookups and doesn't do filesystem searching.

Reply:
Right: `TEXMFAUXTREES=stuff mtxrun --find-file stuff` confirms that the variable works as intended, provided it's added to `texmfcnf.lua` as in the commit, so we are good with ConTeXt.